tmt/utils.py

```diff
@@ -2780,6 +2780,15 @@ class CentOSDistGit(DistGitHandler):
     remote_substring = re.compile(r'redhat/centos')
 
 
+class RHGitlab(DistGitHandler):
```

Comment:
```suggestion
class RedHatGitlab(DistGitHandler):
```

It's just four extra letters :)